python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -401,6 +535,55 @@ def get_entries_iter(
             has_more = bool(next_)
             yield result
 
+    def submit_feedback(
+        self,
+        entry_id: str,
+        feedback_type: NCMECFeedbackType,
+        affirmative: bool,
+        reason_id: t.Optional[str] = None,
+    ) -> GetEntriesResponse:
+        if not affirmative and not reason_id:
+            raise ValueError("Negative feedback must have a reason_id")
```

Comment:
(note to self): If reason map only has one item in it, choose that item. Also test NCMEC API to make sure it won't accept empty reason, which should be our pref otherwise.